### ISSUE TITLE
Documentation contains a command that fails

### ISSUE THREAD

**Issue by pgrange** [n0vember/nocloud#14]
One of the firsts commands in documentation just fails for me with an odd message:

```
#> vmrun -g test -m debian8:5:deb
One (and only one) option must be specified.
usage : vmrun -f <FILE> | -r <GROUP> [ -m specline ] [ -g ]
   or : vmrun -s <GROUP>
  -f <FILE> : instanciate servers based on a configuration file. View documentation for file format.
  -r : do not create new VMs and run the ones that had been previously kept in the specified group
       or those specified using -m option.
  -m : specify a vm specification (as provided to -f option)
  -g : when running VMs have a GUI for each one
  -s : stop a running pf by sending it SIGUSR1 (see NOTE below)
  -b : background mode. Just launcheds VM (with -f or -r) and exit.
  -h : display this help
  NOTE : SIGINT (or ^C) will stop all the managed VMs and will then destoy them. SIGUSR1 will stop and quit as well, but will leave the machiness in place for you to relaunch then at will (that's what the -r option is for).
```

**Comment by n0vember** [comment]
-g is an option to have gui for each vm (console window). Group must be specified with -r to use with -m.

**Comment by n0vember** [comment]
This problem is solved by commit 906207a344f5964f924fc66f2d641a65b4566eef

Thanks for the tip. At least someone reads the documentation :)
